Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -59,11 +60,12 @@ module.default = class ZipScript extends QuickScript {
             .archive(archive)
             .to(wine.prefixDirectory() + "/drive_c/" + this._name)
             .extract();
+        if (fileExists(this._setupPath)){
+	    if (this._setupPath !== 'undefined') {
```

Comment:
The second if is not usefull. If _setupPath is undefined, it will not "exists". You also lost the ability to say to the user that its path is bad if it actually provided one.